src/plugins/console/public/styles/_app.scss

```diff
@@ -19,7 +19,7 @@
 }
 
 .consoleTabs {
-  padding: 0 $euiSizeS;
+  padding: 0 $euiSizeM 0 $euiSizeS;
```

Comment:
Looking at the screenshot, the padding between these icon buttons looks excessive. Can we tighten this up? For reference, see the icon buttons in the top right of Kibana... this represents the more common spacing/pattern.

Reply:
Agree with Ryan, we can shrink the spacing between the button icons. I think the mock has space small (8px) between button elements.

Reply:
Sure, I added `gutterSize: "s"` to the EuiFlexGroup for these icons and this decreased the spacing. The mentioned padding from this class is for the whole top bar, including the menu tabs.

Reply:
Good point. If you're using flex with gutter size, then you may not need these padding styles any longer.

Reply:
I think these padding styles are still needed as they are for the whole top bar, while the gutter size is for the icons flex group only. I did remove the unnecessary padding sizes though.